Extensions/ExternalTfs/Src/vss-extension.json

```diff
@@ -287,6 +126,23 @@
               "maxLength": 300
             }
           },
+          {
+            "id": "definitionSearch",
```

Comment:
based on today's discussion, need to update input descriptor for definition to return definitionSearch as property. 

pl. sync up with Satinder to get this done the right way.

Reply:
Will take care of it.